src/app.js

```diff
@@ -21,5 +21,49 @@ importer.listen(dirWatcher);
 
 // express app
 const app = express();
+app.use(cookieParser());
+
+app.use((req, res, next) => {
+  req.parsedCookies = req.cookies;
+  console.log(req.parsedCookies);
+  next();
+});
+
+app.use((req, res, next) => {
+  req.parsedQuery = req.query;
+  console.log(req.parsedQuery);
+  next();
+});
+
+app.get('/', (req, res) => {
+  res.end('hello express');
+});
+
+app.get('/api/products', (req, res) => {
+  res.send('Returning ALL products');
+  res.end();
+});
+
+app.get('/api/products/:id', (req, res) => {
+  const { id } = req.params;
```

Comment:
If you don't need anything except id you can destructure function params:
`app.get('/api/products/:id', ({ params: { id }}, res) => {`